lib/ansiblelint/formatters/__init__.py

```diff
@@ -26,3 +26,16 @@ def format(self, match):
                                 "E" + match.rule.id,
                                 match.message,
                                 )
+
+from ansiblelint.utils import stringc
+
+class ColoredFormatter(object):
```

Comment:
Rather than using a new formatter, perhaps update the existing formatters to handle colour if it's switched on.